lib/template-block.js

```diff
@@ -0,0 +1,32 @@
+module.exports.register = function (registry, config) {
+const handlebars = require("handlebars")
+  
+// [template,example$foo.json]
+
+  registry.block(function () {
+    var self = this
+    
+    self.named('template')
+    self.positionalAttributes('resourceId')
+    self.onContext(['listing', 'open', 'paragraph'])
```

Comment:
I would recommend only using the open block (and maybe the paragraph). That's because the output becomes an open block. If it were used on a listing as is, you'd be changing the content model, which is not friendly to tooling. This could be resolved by detecting the original content module (by looking for `cloaked-context`), but it would be best to make that a separate enhancement.

Reply:
That's fine, I've previously used only `open` (for the "Markdown-ish" block) but I think I copied this from a doc/example in case it was best practice... will revert to just open 👍